.travis.yml

```diff
@@ -46,7 +43,7 @@ matrix:
           env: CONFIG_OPTS="--debug --coverage no-asm enable-rc5 enable-md2 enable-ec_nistp_64_gcc_128 enable-ssl3 enable-ssl3-method enable-nextprotoneg enable-weak-ssl-ciphers enable-external-tests no-shared -DPEDANTIC -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION" COVERALLS="yes" BORINGSSL_TESTS="yes" CXX="g++-5"
         - os: linux
           compiler: clang-3.6
-          env: CONFIG_OPTS="enable-msan"
```

Comment:
And question was if it's "per definition". I.e. is it in msan's very nature to trigger false positives without no-asm? Point is that if not, then probably it shouldn't be "hardwired" like that in Configure. What about platforms other than x86? Or looking it from the other side. Why don't other sanitizers deserve same treatment? All of them remain blind to assembly modules [and this in fact *is* per definition], so why not enforce no-asm for *all* sanitizers?